### ISSUE TITLE
Obs linux don't load module

### ISSUE THREAD

**Issue by Hideman85** [Palakis/obs-websocket#284]
Same as this issue: https://github.com/Palakis/obs-websocket/issues/58
This is the log file: https://pastebin.com/RRVBBsr1
Install logs:
```
sudo dpkg -i obs-websocket_4.5.0-1_amd64.deb 
(Reading database ... 619984 files and directories currently installed.)
Preparing to unpack obs-websocket_4.5.0-1_amd64.deb ...
Unpacking obs-websocket (4.5.0-1) ...
Setting up obs-websocket (4.5.0-1) ...
```
Found locales in /usr/share/obs/obs-plugins/obs-websocket/
But no way to locate obs-websocket.so

**Comment by Hideman85** [comment]
I found obs-websocket.so in /lib/obs-plugins but it does not loaded!
I found this plugin https://github.com/bazukas/obs-linuxbrowser
I install it like they said in $HOME/.config/obs-studio/plugins/ and it works.
I copy your lib here but nothing, try to make the same directories nothing to, try by rename as libobs-websocket.so nothing too.
I also try to compile from your source but missing `#include <obs-frontend-api.h>` maybe it's the reason that your plugin it's not loaded...
It need to programmatic managed obs, so your plugin it's good for that, but don't work!
